### ISSUE TITLE
Cycles display is wrong for code shown in attachment

### ISSUE THREAD

**Issue by u4ictgrm** [lupino3/edumips64#48]
I patched the diagram shown in the PNG below so my students would grok the idea.

![edumipsbug](https://f.cloud.github.com/assets/7019161/2480470/27fccefe-b0be-11e3-9c03-7aba491e1d1d.png)

Code up the simple loop 

```
      .data
a:    .word 1
b:    .word 0
      .text
go:   ld r1,a(r0)
      daddi r2,r0,2   ; loop counter
      daddi r3,r0,0   ; end value
loop: daddi r2,r2,-1  ; dec counter
      daddi r1,r1,1   ; loop body
      bne r2,r3,loop  ; till done
      sd r1,b(r0)
      halt
```

and run it (either single-step F7 or entirely F4) in EduMIPS64 v1.2

You will see that the final Cycles diagram is wrong as shown below:

![cyclesbug](https://f.cloud.github.com/assets/7019161/2480520/919ee738-b0bf-11e3-9d45-e30db2e7b373.png)

The Execution data in the Statistics window is correct, but the Pipeline window looks like it is doing predict-not-taken.

If it is, in fact using a predict-not-taken policy, then both the Cycles and Statistics window are wrong.  If it is using flush, then the Cycles and Pipeline window are wrong.

I'd actually like to see predict-not-taken and flush as options, but will settle for a consistent view of either across all EduMIPS windows.

Respectfully,
Gerald R. Morris, Ph.D.
JSU Computer Engineering Department

**Comment by lupino3** [comment]
Another diagram of the problem (from an email by prof. Morris)

![screenshot from 2017-12-17 11-32-57](https://user-images.githubusercontent.com/200139/34079102-67185eaa-e31e-11e7-9b07-1c223a781e9e.png)

**Comment by lupino3** [comment]
Fixed in #214.
